pom.xml

```diff
@@ -141,16 +141,17 @@
         <maven.rat.plugin.version>0.12</maven.rat.plugin.version>
         <maven.scala.version>4.4.0</maven.scala.version>
         <maven.scoverage.version>1.3.0</maven.scoverage.version>
-        <maven.shade.version>3.2.1</maven.shade.version>
+        <maven.shade.version>3.1.1</maven.shade.version>
```

Comment:
Can it be updated? @benedeki had a similar question [here](https://github.com/AbsaOSS/enceladus/pull/1739#discussion_r617523468)

Reply:
Yes, probably a merging mistake, fixed, thanks.